compose/smash/nix/nix.conf

```diff
@@ -0,0 +1,10 @@
+#this file attached in volume to docker because we need substituters to be there before nix starts
+
+build-users-group = nixbld
+
+experimental-features = nix-command flakes
```

Comment:
You could (maybe) simplify the setup ceremony with just:

```
FROM alpine:…

RUN apk add --no-cache curl && curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix/tag/v0.11.0 | sh -s -- install linux \
  --extra-conf "sandbox = false" \
  --extra-conf "substituters = https://cache.nixos.org https://cache.iog.io" \
  --extra-conf "trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" \
  --init none --no-confirm
```

This should allow you to run `nix build github:…#…smash-server`

Reply:
I am going to try this; never thought of it.

Reply:
Yeah, I think the official NixOS image is kind of heavy, this is bare bones

Reply:
Works as expected, thanks!

Reply:
@michalrus done in a908181e4f6b6d438a66cb2f8da5844425f3f8dc